src/back/auth/src/api/token_refresh.cpp

```diff
@@ -15,21 +18,27 @@ formats::json::Value TokenRefresh::HandleRequestJsonThrow(
 	const formats::json::Value& body,
 	server::request::RequestContext&) const
 {
-	if (!body.HasMember("token"))
-	{
-		req.SetResponseStatus(server::http::HttpStatus::kBadRequest);
-		return formats::json::FromString(
-			R"({"error": "missing required field 'token'"})");
+
+	formats::json::ValueBuilder res;
+
+	const auto& sessionId = req.GetHeader(headers::kSessionId);
+	if (sessionId.empty()) {
+		res["err"] = "Empty sessionId header";
+		return res.ExtractValue();
 	}
 
-	auto token = body["token"].As<std::string>();
-	auto data = _s.TokenRefresh(token);
+	const std::string userAgent = req.GetHeader(http::headers::kUserAgent);
+
+	try {
+		res = _s.RefreshSession(sessionId, userAgent);
+	}
+	catch(const std::exception& e) {
+		LOG_WARNING() << "Fail to refresh session token: " << e.what();
```

Comment:
Добавь пожалуйста ID сессии:
```suggestion
		LOG_WARNING() << '[' << sessionId << "] Fail to refresh session token: " << e.what();
```

Reply:
Поправил.